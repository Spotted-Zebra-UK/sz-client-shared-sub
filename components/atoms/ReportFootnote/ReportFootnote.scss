.ReportFootnote {
  opacity: 0.54;
}
