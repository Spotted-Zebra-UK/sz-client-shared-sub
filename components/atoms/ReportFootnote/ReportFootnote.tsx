import './ReportFootnote.scss';
import moment from 'moment';
import React, { FC } from 'react';

interface IReportFootnote {
  position: string;
  reportDate: string;
}

const ReportFootnote: FC<IReportFootnote> = props => {
  const { position, reportDate } = props;

  return (
    <div className="ReportFootnote">
      <span className="ReportFootnote__Position">{position}</span>
      <span className="ReportFootnote__Text">
        {' | Candidate report created '}
      </span>
      <span className="ReportFootnote__Date">
        {moment(reportDate, 'x').format('D MMMM YYYY')}
      </span>
    </div>
  );
};

export default ReportFootnote;
