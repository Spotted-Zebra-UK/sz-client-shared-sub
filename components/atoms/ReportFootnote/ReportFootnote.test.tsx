import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import React from 'react';
import ReportFootnote from './ReportFootnote';

describe('ReportFootnote', () => {
  it('should render correctly by default', () => {
    const wrapper = shallow(
      <ReportFootnote position="test-position" reportDate="0" />
    );

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
