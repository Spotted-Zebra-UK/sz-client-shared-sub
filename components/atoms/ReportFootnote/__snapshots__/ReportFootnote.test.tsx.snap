// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReportFootnote should render correctly by default 1`] = `
<div
  className="ReportFootnote"
>
  <span
    className="ReportFootnote__Position"
  >
    test-position
  </span>
  <span
    className="ReportFootnote__Text"
  >
     | Candidate report created 
  </span>
  <span
    className="ReportFootnote__Date"
  >
    1 January 1970
  </span>
</div>
`;
